packages/nextjs/src/edge/transport.ts

```diff
@@ -21,18 +73,16 @@ export function makeEdgeTransport(options: EdgeTransportOptions): Transport {
       ...options.fetchOptions,
     };
 
-    try {
-      return fetch(options.url, requestOptions).then(response => ({
+    return fetch(options.url, requestOptions).then(response => {
+      return {
         statusCode: response.status,
         headers: {
           'x-sentry-rate-limits': response.headers.get('X-Sentry-Rate-Limits'),
           'retry-after': response.headers.get('Retry-After'),
         },
-      }));
-    } catch (e) {
-      return Promise.reject(e);
-    }
+      };
+    });
   }
 
-  return createTransport(options, makeRequest);
+  return createTransport(options, makeRequest, new IsolatedPromiseBuffer(options.bufferSize));
```

Comment:
m: If we export `IsolatedPromiseBuffer`, we can add unit tests for `drain`, I think we should have that to prevent regressions.

Reply:
I'll add some!